app/src/main/java/com/example/playlistmaker/media/ui/MediaFragmentFavoriteTracks.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.playlistmaker.media.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.playlistmaker.databinding.MediaFragmentFavoriteTracksBinding
+
+class MediaFragmentFavoriteTracks : Fragment() {
+
+    private var _binding: MediaFragmentFavoriteTracksBinding? = null
+    private val binding get() = _binding
```

Comment:
При использовании `viewBinding` во фрагментах обязательно нужно обнулять `_binding` в `onDestroyView`, иначе будет утечка памяти. Почитать о том, как использовать эту фичу во фрагментах можно тут - https://developer.android.com/topic/libraries/view-binding#fragments

Reply:
```suggestion
    private val binding get() = _binding!!
```
И ещё вот так нужно написать, иначе смысла заводить переменную binding нет

Reply:
Спасибо огромное за комментарии!!! Упустила этот момент.

Reply:
Нашла еще хорошее объяснение по этой теме: 
https://stackoverflow.com/questions/65295104/android-view-binding-clear-binding-in-fragment-lifecycle

Reply:
Да, написано всё по делу, хоть и без конкретики, но это так - лайфсайкл фрагмента и лайфсайкл вью фрагмента отличаются